### DISCUSSION TITLE
Indexing Initial Directory

### DISCUSSION THREAD

**Discussion by nickisyourfan** [reorproject/reor#65] [General]
First - Super cool app! Excited to see what I can do with this. I started by symlinking a docs directory from a markdown enabled docs site into my reor directory - It's answering my questions really well!

I did make the mistake of choosing my home folder for the root directory... This was a mistake as there were so many other files to index. 

It could be nice to only index markdown files? Or maybe warn the user that the directory chosen has an excess number of files?

Maybe it's just a me problem! 

Thanks for the cool app either way!

**Comment by samlhuillier** [comment]
Thank you for raising this! Yesterday I pushed a release that allows you to open new windows with different vault directories. On indexing, a fix will be coming to allow you to choose file types & restrict/limit to only text types. Sorry about this, I've been rather swarmed and only getting round to solving things now.
